Assets/Scripts/Enemies/FlyingEnemyProjectile.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class FlyingEnemyProjectile : MonoBehaviour
+{
+    [SerializeField] float speed =10f;
+    [SerializeField] float Damge = 10f;
+    Rigidbody2D rb;
+    PlatformerController player;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        player = FindObjectOfType<PlatformerController>();
+        rb = GetComponent<Rigidbody2D>();
+        rb.AddForce((player.transform.position - transform.position).normalized * speed);
+        //destroy itself after 5 seconds
+        StartCoroutine(DestroyProjectile());
+    }
+
+IEnumerator DestroyProjectile()
+    {
+        yield return new WaitForSeconds(5f);
+        Destroy(gameObject);
```

Comment:
[Destroy](https://docs.unity3d.com/ScriptReference/Object.Destroy.html) actually have a timer built into it so you can use that instead of using a coroutine.